shared/js/devtools/panel.es6.js

```diff
@@ -102,31 +139,31 @@ const actionHandlers = {
             cleanUrl.search = ''
             cleanUrl.hash = ''
             cells[1].textContent = cleanUrl.href
-            cells[2].textContent = `🍪 ${action}`
+            cells[2].querySelector('.request-action').textContent = `🍪 ${action}`
             cells[3].textContent = kind
             cells[4].textContent = type
             row.classList.add(kind)
-            table.appendChild(row)
+            return row
```

Comment:
I think it would be simpler to follow calling addRequestRow here are there any downsides to that? The type signatures would be a little less complex.

```suggestion
            return addRequestRow(row)
```

Reply:
@jonathanKingston I've moved the calls to `addRequestRow` to the end of each segment